team-agreements/definition-of-ready/readme.md

```diff
@@ -2,20 +2,24 @@
 
 When the development team picks a user story from the top of the backlog, the user story needs to have enough detail to estimate the work needed to complete the story within the sprint. If it has enough detail to estimate, it is Ready to be developed.
 
-> In case a user story is not Ready in the beginning of the Sprint it increases the chance that your story will not be done at the end of this sprint.
+> If a user story is not Ready in the beginning of the Sprint it increases the chance that the story will not be done at the end of this sprint.
 
 ## What it is
 
-Definition of Ready is the agreement made by the scrum team around how complete a user story should be in order to be selected as candidate for estimation in the sprint planning.
+*Definition of Ready* is the agreement made by the scrum team around how complete a user story should be in order to be selected as candidate for estimation in the sprint planning. These can be codified as a checklist in user stories using [Github Issue Templates](https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository) or [Azure DevOps Work Item Templates](https://docs.microsoft.com/en-us/azure/devops/boards/backlogs/work-item-template?view=azure-devops&tabs=browser).
 
-It can be understood as a checklist that helps the Product Owner to ensure that the user story he wrote contains all the necessary details for the scrum team to understand the work to be done.
+It can be understood as a checklist that helps the Product Owner to ensure that the user story they wrote contains all the necessary details for the scrum team to understand the work to be done.
 
 ### Examples of ready checklist items:
 
 * [ ] Does the description have the details including any input values required to implement the user story?
 * [ ] Does the user story have clear and complete acceptance criteria?
 * [ ] Does the user story address the business need?
 * [ ] Can we measure the acceptance criteria?
+* [ ] Can the story be completed within a single sprint? If not, can it be broken down into smaller stories?

```

Comment:
Despite the fact that verifying if a user story can be completed within a single sprint is a good practice, it wouldn't be a blocker to the user store be considered ready. The sprint planning will be the right place to define this as the feasibility of completing the user story will depend on the team capacity in the sprint. So, I suggest to change by something like this: "Is the user story accomplishing a scope that can not be split into meaningful parts that will attend the customer needs?"

Reply:
I'm trying to think through this and am having a tough time coming up with the correct phrasing. The suggested phrasing:

> Is the user story accomplishing a scope that can not be split into meaningful parts that will attend the customer needs?

Feels very similar to the item a few lines above:

> Does the user story address the business need?

Do you agree? What I am trying to indicate here is that the story should aim to be a minimal slice of functionality that should be completed as part of the same "unit". What are your thoughts on this phrasing, borrowing some phrases from [here](https://martinfowler.com/bliki/UserStory.html)?

> Is the user story small enough to be implemented in a short amount of time, but large enough to provide value to the customer?

Reply:
Good point and I found myself thinking for a while on how to better phrase this question in my first review. The rephrasing you proposed: "Is the user story small enough to be implemented ..." is much better than mine, sure we can keep it. Thanks.